web/src/components/feature/FeedFlowInput.tsx

```diff
@@ -0,0 +1,33 @@
+import { Switch, TextField } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
+const FlexContainer = styled.div`
+  display: flex;
+  gap: 16px;
+`
+
+export const FeedFlowInput = (props: {
+  feedFlow: number
+  feedUnit: string
```

Comment:
```suggestion
  feedUnit: 'kg/d' | 'Sm3/d'
```

Reply:
Was not able to do this without defining a type. Do you approve of this solution?

Reply:
That might actually be even better